simple_use_case/kadistudio/simple_use_case.flow

```diff
@@ -0,0 +1,1199 @@
+{
+    "connections": [
```

Comment:
This file is pretty large in comparison to the workflow descriptions in other languages. You said you have a GUI in which users can graphically create their workflow in order to create this file in a convenient way, right?

Reply:
Exactly. The normal user will never look at the content of the flow file and manipulate it directly but will use the KadiStudio GUI to apply changes to the workflow. In the README there is a link right at the bottom, that links to the visualisation of the workflow file. This visualisation is very similar to the GUI of KadiStudio and gives an idea on how the user can create or manipulate workflows. 
In the demo instance of Kadi4Mat (https://demo-kadi4mat.iam-cms.kit.edu/) a basic workflow editor is also implemented, that can be utilised to get an idea about how workflows can be created.

Reply:
Where exactly do I find the basic workflow editor?

Reply:
I think it would be very important that a third person understands or at least gets a glimpse of how working with Kadi4Mat, KadiStudio respectively would look like. Maybe a screenshot or a GIF would be a good option. Or is it enough to state that the workflow is defined in the GUI. What do you think @dglaeser ?